src/main.qml

```diff
@@ -56,60 +57,90 @@ Application {
+        id: mainRow
+        anchors {
+            verticalCenter: parent.verticalCenter
+            verticalCenterOffset: -parent.height * 0.002
+            left: parent.left
+            leftMargin: parent.width * 0.159
+            right: parent.right
+            rightMargin: parent.width * 0.016
+        }
         height: Dims.h(70)
+
         Spinner {
             id: hourLV
             currentIndex: 0
             enabled: !timer.running
             height: parent.height
-            width: Dims.w(20)
-            model: 10
-            delegate: SpinnerDelegate { text: index }
+            width: parent.width * 0.16
+            model: 24
+            delegate: SpinnerDelegate { text: zeroPad(index) }
             onCurrentIndexChanged: if(enabled) seconds = secondLV.currentIndex + 60*minuteLV.currentIndex + 3600*hourLV.currentIndex
         }
 
-        Label {
-            text: ":"
-            anchors.verticalCenter: parent.verticalCenter
-            horizontalAlignment: Text.AlignHCenter
-            width: Dims.w(20)
-            font.pixelSize: Dims.l(12)
-        }
-
         Spinner {
             id: minuteLV
             currentIndex: 5
             enabled: !timer.running
             height: parent.height
-            width: Dims.w(20)
+            width: parent.width * 0.505
             model: 60
             highlightMoveDuration: currentIndex != 0 ? 400 : 0
             onCurrentIndexChanged: if(enabled) seconds = secondLV.currentIndex + 60*minuteLV.currentIndex + 3600*hourLV.currentIndex
         }
 
-        Label {
-            text: ":"
-            anchors.verticalCenter: parent.verticalCenter
-            horizontalAlignment: Text.AlignHCenter
-            width: Dims.w(20)
-            font.pixelSize: Dims.l(12)
-        }
-
         Spinner {
             id: secondLV
             currentIndex: 0
             enabled: !timer.running
             height: parent.height
-            width: Dims.w(20)
+            width: parent.width * 0.16
             model: 60
             highlightMoveDuration: currentIndex != 0 ? 400 : 0
             onCurrentIndexChanged: if(enabled) seconds = secondLV.currentIndex + 60*minuteLV.currentIndex + 3600*hourLV.currentIndex
         }
     }
 
+    Label {
+        text: "h"
+        anchors {
+            centerIn: app
+            verticalCenterOffset: -app.height * 0.056
+            horizontalCenterOffset: -app.width * 0.199
+        }
+        horizontalAlignment: Text.AlignHCenter
+        font.pixelSize: Dims.l(6.4)
+        font.styleName: "Medium"
+        opacity:0.9
+    }
+
+    Label {
+        text: "m"
+        anchors {
+            centerIn: app
+            verticalCenterOffset: -app.height * 0.056
+            horizontalCenterOffset: app.width * 0.084
+        }
+        horizontalAlignment: Text.AlignHCenter
+        font.pixelSize: Dims.l(6.4)
+        font.styleName: "Medium"
+        opacity:0.9
+    }
+
+    Label {
+        text: "s"
```

Comment:
These label text strings need to be translatable. In German for example i suppose that h should an s ? Or maybe in other languages these need to change.

Reply:
I do not know about other languages then german in that aspect.
But from german point of view, the `HHh MMm SSs` syntax/suffix used for the Stop Watch and now copied into https://github.com/AsteroidOS/asteroid-timer/commit/cecf4f50d3e7efcd61821bcca81a31824160c669 is completely verboten 👮.
DIN 5008 defines we shall only use [HH:MM:SS format](https://de.wikipedia.org/wiki/DIN_5008). 
Even spacings larger then the letter spacing between the `:` dividers are not allowed. 
Sadly the specifically lovely [DIN 5008 is not published freely](https://www.din.de/de/mitwirken/normenausschuesse/nia/veroeffentlichungen/wdc-beuth:din21:318422674) since DIN has copyright on it.
But if you are interested, i could Fax you a copy from my DIN catalog we get at birth. Since personal sharing among friends is allowed according to the license.
Now that we have that out of the way, maybe i can abbreviated to other languages where this format does just not exist officially but is still heavily used in the public, even press and advertising.

To me personally it is a not to be translated, since quite universally understood, artistic representation of time when used in regions that use `HHh MMm SSs` unofficially. 
For german, even DIN refers to "their own" format as `HH:MM:SS`, explicitly using H for the Stunde/hour to not confuse them with the seconds S.
That's how i looked at the Stop Watches format all along. 
It just looks (very) cool but imo does not add too much clutter, which is the whole point of (office related) DIN. Keep clutter at a minimum and represent things perfectly legible and easy to grasp.
In AOS context, the design is kind of unique regarding the layout details and thus my intention was to abbreviate that to the Timer to raise consistency.

But i understand the fact that AOS is about localisation and international community. 
And in case there are other languages that have a commonly used translation for the `HHh MMm SSs` format that ideally does not break the layout, i am all for it.
Is it about wrapping the `h m s` chars into `qsTr("h")`?

Reply:
Ok, fair enough. :) Let's keep this part of the PR as it is then.
Just make sure you don't get into troubles for breaking DIN 5008, this looks very serious! :D 

And yes it would have been about wrapping them with qsTr ;)